lib/consumer.ex

```diff
@@ -113,7 +114,8 @@ defmodule GenRMQ.Consumer do
       deadletter_queue: "gen_rmq_in_queue_error",
       deadletter_queue_options: [
         arguments: [
-          {"x-queue-type", :longstr ,"quorum"},
+          {"x-queue-type", :longstr ,"quorum"}
+        ]
```

Comment:
Is this change meant to be in this PR?

Reply:
Yes, it is a code documentation. There was a typo.